Tests/CareKitEssentialsTests/CareTaskProgressStrategynTests.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  CareTaskProgressStrategyTests.swift
+//  CareTaskProgressStrategyTests
+//
+//  Created by Luis Millan on 7/10/24.
+//  Copyright © 2024 Network Reconnaissance Lab. All rights reserved.
+//
+
+import XCTest
+import CareKitStore
+@testable import CareKitEssentials
+
+private extension OCKAnyEvent {
```

Comment:
Can you place this at the end of the file? The test should be at the top since it's the name of the file